### PR TITLE
Ux/make add photo more clickable

### PR DESCRIPTION
**Pull request by munroTom**
1) https://trello.com/c/FTAA0gzf/88-ux-people-dont-know-that-they-need-to-click-on-the-camera
2) https://trello.com/c/yJh1FETY/107-change-feedback-text
3) https://trello.com/c/HHM5LXHr/114-ux-retake-button-photo-has-the-wrong-color

### CODE REVIEW

File: src/components/PhotoPage/index.js [hunk 1]
```diff
@@ -361,23 +361,23 @@ class PhotoPage extends Component {
...
-            <div style={{display:'flex',flexDirection:'column',flex:1}} className={classes.photo}>
+            <div style={{ display: 'flex', flexDirection: 'column', flex: 1 }} className={classes.photo}>
               <div className='picture'>
-               <img src={this.state.imgSrc} alt={""}/>
+                <img src={this.state.imgSrc} alt={""} />
               </div>
 
               <div className={classes.button}>
-                <Button variant="outlined" fullWidth={true} onClick={this.retakePhoto}>
+                <Button variant="outlined" fullWidth={true} onClick={this.retakePhoto} color="secondary">
```
Comment (munroTom) [inline src/components/PhotoPage/index.js]: 3

File: src/components/WriteFeedbackPage.js [hunk 2]
```diff
@@ -117,82 +117,82 @@ class WriteFeedbackPage extends React.Component {
...
+            autoComplete="email"
+            margin="normal"
+            variant="filled"
+            onChange={this.handleEmailChange}
+            value={this.state.email}
+          />
+          <TextField
+            fullWidth
+            id='feedback-textfield'
+            placeholder='Please write your feedback here'
```
Comment (munroTom) [inline src/components/WriteFeedbackPage.js]: 2

File: src/App.js [hunk 3]
```diff
@@ -266,110 +266,119 @@ class App extends Component {
...
                 ? 'flex'
-                : 'none'
-              }}
-            >
-              <AddAPhotoIcon />
-            </Fab>
-          </main>
+                : 'none',
+
+              borderRadius: '10px',
```
Comment (munroTom) [inline src/App.js]: 1, I had a quick look and couldn't see a text prop here
Screenshot 2019-07-16 at 19 51 19

File: src/App.js [hunk 4]
```diff
@@ -43,13 +43,13 @@ const styles = theme => ({
   },
   camera: {
     position: 'absolute',
-    bottom: theme.spacing.unit * 2,
-    right: theme.spacing.unit * 2
+    bottom: theme.spacing.unit,
```
Comment (munroTom) [inline src/App.js]: Also reduced the padding here with as part of 1

**Comment by sebastianovide** [comment]
Hi @munroTom 

thanks for the PR. Question for you ? is plastic patrol "breaking free" from Photos (https://github.com/Geovation/photos) ?

yes means less over head as you need to update only PP project. But it also means that you will not get updates from photos anymore.
no means that you will get updates from photos. But it also means more overhead: you will need to push into photos  and then synchronize the two projects.

FYI: @globalavocado

**Comment by munroTom** [comment]
Hi @sebastianovide 

Probably not a decision for me! If the changes I've made mean that has to happen then it's worth having a discussion with @globalavocado, Lizzy and Liz. 
I can update the changes to mean it doesn't eject, just let me know

**Comment by sebastianovide** [comment]
@munroTom I'll be happy to marge, but there are some conflicts to fix first

**Comment by munroTom** [comment]
@sebastianovide conflicts should all be resolved now

**Pull request merged**